pwa/src/components/translations/translationTableForm/translationTableForm.css

```diff
@@ -0,0 +1,3 @@
+.h5-translationForm {
+  font-size: 20px;  
+}
```

Comment:
I would suggest the following: because the component we're in is called `TranslationTableForm` I would prefix the `css` class with `TranslationTableForm-`. So the `h5` would become `.TranslationTableForm-h5`. This way (if implemented consistent) you can be sure to never write css rules that are targeting unwanted elements.

Missing a new line at the end of this file.